### PR TITLE
feat(logging)!: drop `logFile` and `logFileLevel`

### PR DESCRIPTION
**Pull request by RahulGautamSingh**
## Changes
- Drop `logFile` and `logFileLevel`

## Context
Closes: https://github.com/renovatebot/renovate/issues/14705

## Documentation (please check one with an [x])

- [x] I have updated the documentation, or
- [ ] No documentation update is required

## How I've tested my work (please select one)

I have verified these changes via:

- [ ] Code inspection only, or
- [x] Newly added/modified unit tests, or
- [ ] No unit tests but ran on a real repository, or
- [ ] Both unit tests + ran on a real repository

### CODE REVIEW

**Review by rarkins — APPROVED** [review]

File: lib/config/validation.ts
Comment (viceice) [inline lib/config/validation.ts:892]: why this? can we use istanbul ignore here for future use?

**Review by viceice — COMMENTED** [review]
otherwise LGTM

File: lib/config/validation.ts
Comment (rarkins) [reply lib/config/validation.ts:892]: Good point

File: lib/workers/global/config/parse/index.spec.ts
Comment (HonkingGoose) [inline lib/workers/global/config/parse/index.spec.ts:176]:
```suggestion
    it('only initializes the file when the env var LOG_FILE is properly set', async () => {
```

Putting our point positively is easier to read and understand. Check for technical accuracy before applying. :wink: 

Or alternatively, fix the spelling and grammar, and keep the sentence structure:

`it('does not initialize file when the env var LOG_FILE is undefined', async () => {`

**Review by HonkingGoose — CHANGES_REQUESTED** [review]

**Review by HonkingGoose — COMMENTED** [review]
Are you sure you got all references to the things you're removing?

The Jest test description is good now. :smile: 

As usual, I'll let a maintainer do the full review.

**Comment by RahulGautamSingh** [comment]
> Are you sure you got all references to the things you're removing?

Yes. 🤗

**Pull request merged**
